packages/css/src/components/label/label.template.ts

```diff
@@ -13,14 +14,22 @@ const statusMap = new Map<string, string>([
   ['danger', 'Gevaar']
 ]);
 
-export function labelTemplate({ status, label, button, compact }: Label) {
+export function labelTemplate({ status, label, button, compact, symbol }: Label) {
   return html`
-    <span class="dso-label ${classMap({ [`dso-label-${status}`]: !!status, [`dso-label-compact`]: !!compact })}">
-      ${status && html`
-        <span class="sr-only">${statusMap.get(status)}: </span>
-      `}
-      ${label}
-      ${button
+    <span class="dso-label ${classMap({ [`dso-label-${status}`]: !!status, [`dso-compact`]: !!compact })}">
```

Comment:
`dso-label-compact` is fout, de class is `dso-compact`. Dit werkte niet in Storybook

Reply:
Ah ik zie het, was inderdaad mis in het HTML/CSS storybook:
![image](https://user-images.githubusercontent.com/2689559/142596260-ab41707c-2614-4f4c-b021-ce12a7ca1185.png)